File: beginners_selection/abc088b.py
# https://atcoder.jp/contests/abs/tasks/abc088_b

n = int(input())
a_list = sorted(list(map(int, input().split())), reverse=True)

alice_hands = []
bob_hands = []

for a in a_list:
    if len(alice_hands) == len(bob_hands):
        alice_hands.append(a)
    else:
        bob_hands.append(a)

print(sum(alice_hands) - sum(bob_hands))

File: cpp/beginners_selection/abc088b.cpp
// https://atcoder.jp/contests/abs/tasks/abc088_b

#include<iostream>
using namespace std;

int main() {
    int n;
    cin >> n;

    int a[n];
    for (int i=0; i<n; i++) {
        cin >> a[i];
    }

    sort(a, a + n, greater<>());

    int alice = 0;
    int bob = 0;
    for (int i=0; i<n; i++) {
        if (i % 2 == 0) {
            alice += a[i];
        } else {
            bob += a[i];
        }
    }

    cout << (alice - bob) << endl;
    return 0;
}
